src/tree.rs

```diff
@@ -32,7 +33,7 @@ pub trait LayoutTree {
     fn layout_mut(&mut self, node: Node) -> &mut Layout;
 
     /// Mark a node as finished
```

Comment:
This doc comment isn't actually correct, right? 
```suggestion
    /// Marking a node as dirty will tell Taffy that it needs to be recomputed.
```

Reply:
Too late, but this is what I wanted to have changed ^